src/server/scripts/Commands/cs_reset.cpp

```diff
@@ -54,6 +56,44 @@ class reset_commandscript : public CommandScript
         return commandTable;
     }
 
+    static bool HandleResetArenaSeasonCommand(ChatHandler* /*handler*/, const char* args)
+    {
+        if (!sWorld->getBoolConfig(CONFIG_ARENA_SEASON_IN_PROGRESS))
+        {
+            SQLTransaction trans = CharacterDatabase.BeginTransaction();
+
+            std::string param = args;
+            if (*args && param == "delete")
+            {
+                trans->Append("TRUNCATE `arena_team`");
+                trans->Append("TRUNCATE `arena_team_member`");
+                trans->Append("TRUNCATE `character_arena_stats`");
+            }
+            else
+            {
+                trans->PAppend("UPDATE `arena_team` SET `rating`=%u,`seasonGames`=0,`seasonWins`=0,`weekGames`=0,`weekWins`=0,`rank`=0", sWorld->getIntConfig(CONFIG_ARENA_START_RATING));
+                trans->PAppend("UPDATE `arena_team_member` SET `personalRating`=%u,`weekGames`=0,`weekWins`=0,`seasonGames`=0,`seasonWins`=0", sWorld->getIntConfig(CONFIG_ARENA_START_PERSONAL_RATING));
+                trans->PAppend("UPDATE `character_arena_stats` SET `matchMakerRating`=%u", sWorld->getIntConfig(CONFIG_ARENA_START_MATCHMAKER_RATING));
+            }
+            CharacterDatabase.CommitTransaction(trans);
+        }
+        return true;
+    }
+
+    static bool HandleResetHonorWorldCommand(ChatHandler* /*handler*/, const char* args)
+    {
+        SQLTransaction trans = CharacterDatabase.BeginTransaction();
+        trans->PAppend("UPDATE `characters` SET `totalHonorPoints`=%u,`todayHonorPoints`=0,`yesterdayHonorPoints`=0", sWorld->getIntConfig(CONFIG_START_HONOR_POINTS));
+
+        std::string param = args;
+        if (*args && param == "kills")
```

Comment:
please use http://www.cplusplus.com/reference/cstring/strcmp/ to compare const char* strings